### PR TITLE
#7459 - xlsx commands reviewed and updated

### PR DESCRIPTION
**Pull request by Vegebutcher**
one .cs filename changed to reflect the real command name (CountRows)

### CODE REVIEW

File: G1ANT.Addon.Xlsx/G1ANT.Addon.Xlsx/Commands/XlsxCloseCommand.md [hunk 1]
```diff
@@ -1,50 +1,35 @@
...
-**Example 2:**
-
-Here document with .xlsx file extension with ID '444' is being closed.
+In the script below, two Excel files are opened (be sure to provide real filepaths) and their IDs are stored in variables specified by the `result` argument. Then the second file is closed — if its ID wasn’t specified, the `xlsx.close` command would close the first file:
 
 ```G1ANT
-xlsx.close id ♥444
+xlsx.open path C:\Tests\Book1.xlsx result ♥xlsId1
+xlsx.open path C:\Tests\Book2.xls result ♥xlsId2
+xlsx.close id ♥xlsId2
```
Comment (DanielWilczek) [inline G1ANT.Addon.Xlsx/G1ANT.Addon.Xlsx/Commands/XlsxCloseCommand.md]: xlsx.open path ♥environment⟦TEMP⟧\Book1.xlsx createifnotexist true result ♥xlsId1
xlsx.open path ♥environment⟦TEMP⟧\Book2.xls createifnotexist true result ♥xlsId2
xlsx.close id ♥xlsId2
xlsx.close id ♥xlsId1

Comment (DanielWilczek) [reply G1ANT.Addon.Xlsx/G1ANT.Addon.Xlsx/Commands/XlsxCloseCommand.md]: If we use ♥environment⟦TEMP⟧ in examples its going to work just after copy paste.

File: G1ANT.Addon.Xlsx/G1ANT.Addon.Xlsx/Commands/XlsxFindCommand.md [hunk 2]
```diff
@@ -1,46 +1,38 @@
...
-xlsx.open path ‴C:\Tests\Book1.xlsx‴ result ♥xlsHandle
-xlsx.setsheet name ‴Sheet1‴
-xlsx.find value ‴aaa‴ result ♥number errormessage ‴Value not found‴
-dialog ♥number
-xlsx.close id ♥xlsHandle
+xlsx.open C:\Tests\Book1.xlsx
+xlsx.find aaa errorjump notFound
+dialog ‴Value found in the cell: column ♥resultcolumn, row ♥resultrow‴
+label notFound
+dialog ‴Value not found‴
```
Comment (WiktoriaPrusik) [inline G1ANT.Addon.Xlsx/G1ANT.Addon.Xlsx/Commands/XlsxFindCommand.md]: If the value is found, the "Value not found" message will appear anyway so this example should be changed

Comment (Vegebutcher) [reply G1ANT.Addon.Xlsx/G1ANT.Addon.Xlsx/Commands/XlsxFindCommand.md]: corrected

**Pull request merged**